samples/csharp_dotnetcore/12.customQABot/README.md

```diff
@@ -0,0 +1,88 @@
+
+# Prerequisites
+- Create a [Language resource](https://aka.ms/create-language-resource) with Custom question answering enabled.
+- Follow instructions [here][Quickstart] to create a Custom question answering project. You will need this project's name to be used as `ProjectName` in [appsettings.json](appsettings.json).
+
+### Connect your bot to the project.
+Follow these steps to update [appsettings.json](appsettings.json).
+- In the [Azure Portal][Azure], go to your resource.
+- Go to `Keys and Endpoint` under Resource Management.
+- Copy one of the keys as value of `LanguageEndpointKey` and Endpoint as value of `LanguageEndpointHostName` in [appsettings.json](appsettings.json).
+- `ProjectName` is the name of the project created in [Language Studio][LS].
+
+# To try this sample
+
+- Install the Bot Framework Emulator version 4.3.0 or greater from [here][BFE]
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-samples.git
+    ```
+
+- In a terminal, navigate to `samples/csharp_dotnetcore/48.customQABot-all-features`
+- Run the bot from a terminal or from Visual Studio, choose option A or B.
+
+  A) From a terminal
+
+  ```bash
+  # run the bot
+  dotnet run
+  ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `samples/csharp_dotnetcore/48.customQABot-all-features` folder
+  - Select `CustomQABotAllFeatures.csproj` file
+  - Press `F5` to run the project
+- Connect to the bot using Bot Framework Emulator
+  1) Launch Bot Framework Emulator
+  2) File -> Open Bot
+  3) Enter a Bot URL of `http://localhost:3978/api/messages`
+
+# Try Precise Answering
+- Try the following queries:
+  1) Accessibility
+  2) Register
+- You can notice a short answer returned along with a long answer.
+- If testing in [Language Studio][LS], you might have to check `Include short answer response` at the top.
+- You can disable precise answering by setting `EnablePreciseAnswer` to false in [appsettings.json](appsettings.json).
+- You can set `DisplayPreciseAnswerOnly` in [appsettings.json](appsettings.json) to true to display just precise answers in the response.
+
+# Deploy the bot to Azure
+See [Deploy your C# bot to Azure][50] for instructions.
+
+The deployment process assumes you have an account on Microsoft Azure and are able to log into the [Microsoft Azure Portal][Azure].
+
+If you are new to Microsoft Azure, please refer to [Getting started with Azure][70] for guidance on how to get started on Azure.
+
+## Further reading
+
+- [Bot Framework Documentation](https://docs.botframework.com)
```

Comment:
Consider reducing the list of references in this section to only a few key ones to simplify maintenance and avoid broken links in the future.